javascript/javascript1/week3/ItemArray.js

```diff
@@ -0,0 +1,19 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+  // Write some code here
+ names.splice(1,1);
```

Comment:
What if "Ahmad" would be not at the 1-st place ? 
Imagine that list of names is provided by users, so we as developers dont  know position  of "Ahmad" name, and we don't event know if  it's present it the list at all.